disco/api/client.py

```diff
@@ -288,7 +288,34 @@ def guilds_modify(self, guild, reason=None, **kwargs):
         return Guild.create(self.client, r.json())
 
     def guilds_delete(self, guild):
-        r = self.http(Routes.GUILDS_DELETE, dict(guild=guild))
+        self.http(Routes.GUILDS_DELETE, dict(guild=guild))
+
+    def guilds_create(
+            self,
+            name,
+            region=None,
+            icon=None,
+            verification_level=None,
+            default_message_notifications=None,
+            explicit_content_filter=None,
+            roles=[],
```

Comment:
for safeties sake lets default these to `None` and change lines 306 and 307 to do `(roles or [])` and `(channels or [])`.